### DISCUSSION TITLE
Deployment to Vercel

### DISCUSSION THREAD

**Discussion by wanjas** [prisma/prisma#21485] [Q&A] [answered]
### Question

Hello,

out of nowhere I've started to get `/bin/sh: hostname: command not found` error on prisma generate. 

Does anyone encountered the same issue? 

Prisma version 5.1.1, 5.4.2. It's doesn't look like prisma version but looks like some Vercel changes.

**Comment by janpio** [comment]
When does this happen? During the build phase on Vercel?

**Reply by wanjas** [reply]
Yes, during the build phase. It suspiciously happened after this announcement from Vercel [October 11, 2023](https://vercel.com/changelog/strengthening-vercels-infrastructure-against-http-2-rapid-reset-attacks)

I have two effectively identical builds on the 6th and 13th of October. First succeeded, second failed. Updating Prisma to 5.4.2 hasn't helped.

**Reply by janpio** [reply]
Hmm, all our ecosystem-tests for Vercel (and we have a dozen) are passing as usual: https://github.com/prisma/ecosystem-tests

Can you describe roughly your project setup? Or even link to a repo that shows it?

**Reply by wanjas** [reply, marked as answer]
Very sorry for taking your time, it was a combination of `nx` warning marked as an error with insane error messaging caused by somebody localizing the field in Builder.io without any reason. 

Thank you for your attempt to help me. Confirming that it's not a prisma-related thing took me to the very long but right path.

**Reply by janpio** [reply]
All good, thanks for the upadte and great you figured it out!

**Reply by cbrinicombe13** [reply]
@wanjas what was the solution here? Coming from another thread on Vercel where this is an issue: https://github.com/nrwl/nx/issues/22686

**Reply by wanjas** [reply]
@cbrinicombe13 I have no solution. Because I haven't noticed any issues caused by this warning, so I opted to ignore it. 

`/bin/sh: hostname: command not found` wasn't the reason my build crashed.
